src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsKruxObject.java

```diff
@@ -9,7 +9,7 @@
 
 public class AdsKruxObject extends AdsBaseObject {
 
-  private final String kruxCdn = urlBuilder.getUrlProtocol() + "cdn.krxd.net/";
+  private final String kruxCdn = urlBuilder.getUrlProtocol() + "://cdn.krxd.net/";
   private final String kruxControlTagUrlPrefix = kruxCdn + "controltag?confid=";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)